packaging/docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM centos:7
+LABEL maintainer="tdockendorf@osc.edu; johrstrom@osc.edu"
+
+# setup the ondemand repositories
+COPY files/ondemand-web.repo /etc/yum.repos.d/ondemand-web.repo
+COPY files/ondemand-ci-web.repo /etc/yum.repos.d/ondemand-ci-web.repo
+RUN curl https://yum.osc.edu/ondemand/RPM-GPG-KEY-ondemand -o /etc/pki/rpm-gpg/RPM-GPG-KEY-ondemand
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-ondemand
+
+# update
+RUN yum install -y centos-release-scl 
+RUN yum -y update
+
+# install all the dependencies
+RUN yum install -y \
+        file \
+        lsof \
+        sqlite-devel \
+        sudo \
+        httpd24 \
+        rh-ruby25 \
+        rh-nodejs10 \
+        ondemand-passenger-6.0.4-2.el7.x86_64 \
+        ondemand-nginx-1.17.3-2.p6.0.4.el7.x86_64
+
+RUN yum clean all && rm -rf /var/cache/yum/*
+RUN rm -rf /var/cache/yum/*
```

Comment:
Per other comments, these won't actually save any space in final container since the layer with `yum` will contain the cache. This would only be useful if executed on same `RUN` as all other `yum` commands or if executed at end of build script that is executed via `RUN`.